custom/icds_reports/models/util.py

```diff
@@ -69,3 +72,74 @@ class AggregationRecord(models.Model):
     run_date = models.DateField(auto_now_add=True)
     state_ids = ArrayField(models.CharField(max_length=255), null=True)
     agg_uuid = models.UUIDField(unique=True, default=uuid.uuid4)
+
+
+class UcrReconciliationStatus(models.Model):
+    XMLNS = 'xmlns'
+    CASE_TYPE = 'case_type'
+
+    db_alias = models.CharField(max_length=10)
+    day = models.DateField()
+    table_id = models.TextField()
+    doc_type = models.CharField(
+        max_length=10,
+        choices=(
+            (CASE_TYPE, 'Case Type'),
+            (XMLNS, 'XMLNS'),
+        )
+    )
+    doc_type_filter = models.TextField(help_text="Either an XMLNS or case type")
+    last_processed_date = models.DateTimeField(null=True)
+    verified_date = models.DateTimeField(null=True)
+
+    @property
+    def is_form_ucr(self):
+        return self.doc_type == self.XMLNS
+
+    @classmethod
+    def setup_days_records(cls, day):
+        UCR_MAPPING = (
```

Comment:
Is it worth pulling this to a constant, somewhere?

Reply:
I don't see a reason to do that at this time. This is still used as a constant, but it is scoped to the only place it is used. If it were expected to be useful elsewhere, I could see pullling it to a constant file